rocketc/api_teams.py

```diff
@@ -58,3 +58,13 @@ def get_user_team(self, course_id, username):
         if team_request.status_code == 200:
             return team_request.json()["results"]
         return team_request.json()
+
+    def get_members(self, team_id):
+        """Get the team members"""
+        url_path = "team_membership"
+        payload = {"team_id": team_id}
+        team_request = self._call_api_get(url_path, payload)
+
+        if team_request.status_code == 200:
```

Comment:
Question here, what happen if the status code is not 200? it will return the json response. You use this method in rocketc.py line 484. In line 485 you ask if the response is not falsy. As here you're returning a json, the response will pass the check and will try to iterate over the json response that does not contain results (because we are supposing we got an error from API). will it generate an exception right? what do you think about returning None or empty array here when the http request is not successful and logging an error message like "An error has ocurred trying to fetch team members"?

Reply:
you are right this generates a exception, it is better if returns None

Reply:
just fix that one and I would say merge